fiftyone/plugins/utils.py

```diff
@@ -5,14 +5,17 @@
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
+from bson import json_util
 import logging
 import multiprocessing.dummy
 import os
 
 from bs4 import BeautifulSoup
+import json
 import yaml
 
 import fiftyone.core.utils as fou
+from fiftyone.operators import types
```

Comment:
These are all operator-specific utilities, so if we go ahead with any of them, I think they belong in `fiftyone.operators.utils` instead.

Reply:
Please do not create a utils module, instead as I've suggested elsewhere, update the `ExecutionContext` and `types` following the same patterns that already exist.

Reply:
I think that `set_view()` in particular may in fact be best suited on `ExecutionContext`, but potentially there could be other things that are not important enough to clutter the `ExecutionContext` but still merit inclusion in a util file somewhere